src/lyapunovspectrum.jl

```diff
@@ -157,18 +182,22 @@ function liap_taylorstep!(f, t::Taylor1{T}, x::Vector{Taylor1{U}}, dx::Vector{Ta
 end
 
 """
-    liap_taylorinteg(f, q0, t0, tmax, order, abstol; maxsteps::Int=500)
+    lyap_taylorinteg(f!, q0, t0, tmax, order, abstol[, f!]; maxsteps::Int=500)
 
-Similar to [`taylorinteg!`](@ref) for the calculation of the Liapunov
+Similar to [`taylorinteg!`](@ref) for the calculation of the Lyapunov
 spectrum. Note that the number of `TaylorN` variables should be set
 previously by the user (e.g., by means of `TaylorSeries.set_variables`) and
 should be equal to the length of the vector of initial conditions `q0`.
 Otherwise, whenever `length(q0) != TaylorSeries.get_numvars()`, then
-`liap_taylorinteg` throws an `AssertionError`.
+`lyap_taylorinteg` throws an `AssertionError`.  Optionally, the user may provide
+an Jacobian function `f!` to evaluate the current value of the Jacobian.
+Otherwise, the current value of the Jacobian is computed via automatic
+differentiation using `TaylorSeries.jl`. For more details on `f!`, see
+[`lyap_jetcoeffs!`](@ref).
 
 """
-function liap_taylorinteg(f, q0::Array{U,1}, t0::T, tmax::T,
-        order::Int, abstol::T; maxsteps::Int=500) where {T<:Real, U<:Number}
+function lyap_taylorinteg(f!, q0::Array{U,1}, t0::T, tmax::T,
```

Comment:
Same comment as above about changing `Nothing` for some function that reflects the current behavior.

Reply:
Also, instead of calling this optional parameter `df!`, I think something like `jacobian` makes clearer its role.

Reply:
> Same comment as above about changing `Nothing` for some function that reflects the current behavior.

Thanks for the suggestion, and I agree that might be better! I'll work on this once agree on a final implementation for the changes introduced on this PR

> Also, instead of calling this optional parameter df!, I think something like jacobian makes clearer its role.

Sure, thanks! I'll change that

Reply:
See comment above about the name of that function.